test/e2e/fixtures/lint/file-with-prettier-lint-errors.ts

```diff
@@ -1,5 +1 @@
-export const foo  = (   ) => 
-!! ('bar')
-;
-
- 
\ No newline at end of file
+export const foo = () => !!'bar';
```

Comment:
the fixtures is used for e2e testing - should not be changed?

Reply:
Yeah i think the changes we made shouldn't affect e2e tests